src/modules/fw_pos_control/FixedwingPositionControl.hpp

```diff
@@ -825,6 +831,21 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 	void initializeAutoLanding(const hrt_abstime &now, const position_setpoint_s &pos_sp_prev,
 				   const float land_point_alt, const Vector2f &local_position, const Vector2f &local_land_point);
 
+	/*
+	 * Checks if the vehicle satisfies conditions for low-height flight
+	 *
+	 * @return bool True if conditions are satisfied, false otherwise
+	 */
+	bool checkLowHeightConditions();
+
+	/*
+	 * Updates TECS altitude time constant according to the is_low_height parameter.
+	 *
+	 * @param is_low_height Boolean flag defining whether we are in low-height flight
+	 * @param dt Update time step [s]
+	 */
```

Comment:
```suggestion
```
`void updateTECSAltitudeTimeConstant(const bool is_low_height, const float dt);` reads as "function that updates TECS altitude time constant based on is_low_height and dt" to me. The only thing relevant in this comment is maybe the unit of `dt`, but if not specified, I would anyway expect that dt is in seconds.

Reply:
I'll keep this one to stay aligned with the rest of documented methods on this file.